packages/common/src/intl/IntlProvider.tsx

```diff
@@ -13,8 +13,8 @@ export const IntlProvider: React.FC = ({ children }) => {
     const expirationTime =
       process.env['NODE_ENV'] === 'development'
         ? 0
-        : 7 * 24 * 60 * minuteInMilliseconds;
-
+        : // TODO: change back to a week when things are stable
```

Comment:
because the recent changes on the demo server don't have the translations showing yet.
thought that while we are changing often, an hour is probs enough of a balance between improving performance and actually showing the right words?

Reply:
Yeah certainly